### DISCUSSION TITLE
zombies can't get off the bridge

### DISCUSSION THREAD

**Discussion by satarics** [cataclysmbn/Cataclysm-BN#2302] [General]
zombies can't get off the bridge
![image](https://user-images.githubusercontent.com/75517670/213927136-9e40a20d-9d43-4dea-b8f5-9c4c8ac51c63.png)

**Comment by scarf005** [comment]
hello @satarics, thank you for reaching out! however, it's not clear to us how this bug could be reproduced.

I'll close this discussion since this would be more appropriately reported as a bug issue. Could you please follow [the steps to Submit a bug report on github](https://github.com/cataclysmbnteam/Cataclysm-BN#ive-found-a-bug-what-should-i-do) and attach relevent save file so we can investigate the issue more efficiently.

thanks in advance.
